### DISCUSSION TITLE
Problemas com o JS no meu projeto Angular

### DISCUSSION THREAD

**Discussion by venceslaufelipe** [frontendbr/forum#1438] [Perguntas e Respostas]
Olá, pessoal!

Estou com problema quando tento abrir esse componente no meu projeto angular, mas ele funciona normalmente quando abro o arquivo pelo navegador. 

Tem algum código errado? Os scripts que importei não funcionam em Angular? Me ajudem! kkk

_Código: https://gist.github.com/venceslaufelipe/ae9da6bb54de560bf6c93e2bbff5c85a_

Obrigado pela atenção!

**Comment by felipefialho** [comment]
@venceslaufelipe dei uma olhada na código mas não entendi a dúvida
